e2e/testdata/plugin-usage.golden

```diff
@@ -1,23 +1,23 @@
 
 Usage:	docker app [OPTIONS] COMMAND
 
-A tool to build and manage Docker Applications.
+A tool to build, share and run a Docker App
 
 Options:
       --version   Print version information
 
 Management Commands:
-  image       Manage application images
+  image       Manage App Images
 
 Commands:
-  build       Build service images for the application
-  init        Initialize Docker Application definition
-  ls          List the installations and their last known installation result
-  pull        Pull an application package from a registry
-  push        Push an application package to a registry
-  rm          Remove an application
-  run         Run an application
-  update      Update a running application
-  validate    Checks the rendered application is syntactically correct
+  build       Build an App Image from a .dockerapp App Definition
+  init        Initialize an App Definition
+  ls          List Running Apps
```

Comment:
Do we want to capitalise Running and Image?

Note that image is not capitalised in the CLI:
```console
$ docker image help

Usage:	docker image COMMAND

Manage images
```

Reply:
Discussed with Chris. We stick to capitalized for now.